### PR TITLE
Fix PCA plot problem

### PR DESCRIPTION
**Pull request by UxxUnet**

### CODE REVIEW

File: GWAS/PCA.ipynb [hunk 1]
```diff
@@ -1153,13 +1151,8 @@
...
     "    ###\n",
     "    # Get the min and max values for x and y-axes\n",
-    "    if (${\"T\" if len(min_axis) == 0 or len(max_axis) == 0 else \"F\"}) {\n",
-    "        min_axis <- round(colMins(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)\n",
-    "        max_axis <- round(colMaxs(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)  \n",
-    "    } else {\n",
-    "        min_axis <- ${min_axis}\n",
-    "        max_axis <- ${max_axis}\n",
-    "    }\n",
+    "    min_axis <- round(colMins(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)\n",
```
Comment (gaow) [inline GWAS/PCA.ipynb:1154]: @UxxUnet why are we doing this? sometimes we do want to be able to specify min and max axies and not let that depend on data

File: GWAS/PCA.ipynb [hunk 2]
```diff
@@ -1182,8 +1175,8 @@
     "              scale_color_manual(values=color_list) +\n",
     "              theme_classic()\n",
     "        }}\n",
-    "    unit = 4\n",
-    "    n_col = min(4, k)\n",
+    "    unit = 2\n",
```
Comment (gaow) [inline GWAS/PCA.ipynb:1178]: @UxxUnet What's the issue with displaying 4 images per row?

File: GWAS/PCA.ipynb (hunk 1 above)
Comment (gaow) [reply GWAS/PCA.ipynb:1154]: By default, min_axis and max_axis are not specified, right? they are default to empty `""`

File: GWAS/PCA.ipynb (hunk 2 above)
Comment (gaow) [reply GWAS/PCA.ipynb:1178]: If we are trying to decide between 2 or 4 why don't we make it an option? 

```
parameter: n_col = 4
R: ...
    ...
    n_col = min(${n_col}, k)

```

does it make sense?

**Comment by gaow** [comment]
@UxxUnet please move the `PCA_problem.iypnb` to `prototype` folder if you want to keep it. Every file outside that folder should be pipelines not analysis notebooks.

File: GWAS/PCA.ipynb (hunk 1 above)
Comment (dianacornejo) [reply GWAS/PCA.ipynb:1154]: What we want is that when the min_axis and max_axis are set to 0 or none the code defaults to the firs part
```
 if (${"TRUE" if len(min_axis) == 0 or len(max_axis) == 0 else "FALSE"}) {
        min_axis <- round(colMins(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)
        max_axis <- round(colMaxs(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)  
    } else {
        min_axis <- ${min_axis}
        max_axis <- ${max_axis}
```
That behavior is currently not happening. And that's what we need to fix

**Pull request closed without merging**